services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -457,7 +455,10 @@ meters in flow rules at all (#flowType flow)"() {
 
         when: "Resetting meter burst and rate to default"
         northbound.resetMeters(flow.id)
-//        TODO(andriidovhan) add then and check response: body or message
```

Comment:
please, add an empty line between the `when` and `then` blocks

Reply:
Resolved